news/2024/2024-10-09-osulazer-updates-october-9.md

```diff
@@ -0,0 +1,65 @@
+
+![](/wiki/shared/news/2024-10-09-osulazer-updates-october-9/banner.jpg)
+
+The developers are working out the kinks and preparing for server-side release of new pp rework, but they've also been busy with other aspects of osu!(lazer), which you can learn all about either in text form, or by checking out the latter portion of the video digest below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Kqu4TUOO5IY?si=QC-NhQ_Kv99IMZW9&amp;start=333" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Editor changes
+
+- **The edit setup screen has been completely redesigned.** The setup screen has been rebuilt from the ground up with a more stylish look, and a lot more options that mappers will find very convenient.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-10-09-editor-design-comparison.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Objects can now be added to an existing selection when control clicking them, and the selected hit objects can be scaled around the select box centre.** Another set of smaller, intuitive updates that will make mappers' lives easier.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-10-09-editor-selection.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **The grid placement tool has been added.** Next to that, the rotate and flip commands will take into account the centre of the newly placed grid, and grids can be centred around specific hit objects.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-10-09-grid-placement-tool.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Hotkey hints have been added across editor menus.** Very helpful for aspiring mappers.
+
+![](/wiki/shared/news/2024-10-09-osulazer-updates-october-9/hotkeys.png)
+
+## Skinning changes
+
+- **The flexibility of the skin layout editor player avatar has been improved.** Next to being able to round the edges, you can now change the aspect ratio of the element, and the avatar will automatically adapt to the new ratio.
+- **Certain elements can now have their colour customised.** The default and argon song progress bar, font adjustable components, argon wedge piece and the box element all now have adjustable colours, hopefully this is just a start of the editor elements receiving more customisation features.
+
+![image](/wiki/shared/news/2024-10-09-osulazer-updates-october-9/skin-element-colour-picker.jpg)
+
+## Other changes
+
+- **It's now easier to differentiate osu!(lazer) from osu!(stable) in Windows.** Going forward it'll be clearer to players which version of the games they should use to open osu! related files.
```

Comment:
needs elaboration

```suggestion
- **It's now easier to differentiate osu!(lazer) from osu!(stable) in Windows.** In certain context menus and settings (such as when changing the default app to open `.osz` files with), the lazer version is now denoted as "osu!(lazer)" instead of just "osu!".
```